### ISSUE TITLE
Update Element: Postsecondary Program Level 

### ISSUE THREAD

**Issue by CaliStringer** [CEDStandards/CEDS-Elements#301]
This is for capturing needs not currently supported by the CEDS model.  Please do not send or share actual data as examples in this issue or in attachments. 

**Author(s)**
Cali Stringer 

**Authoring Organization(s)**
PS Organization Committee 

**Email address**
Cstringer@crocusllc.com

**Use Case Title**
Proposed Update to Program Level 

**Use Case Description** 
Update to include programs and credentials 

**Use Case Background**
Current Program Levels do not allow for program levels including credentials. 

**Location of Element in the Domain Entity Schema**
Postsecondary

**Comment by angelajubinvilleqip** [comment]
@CaliStringer can you provide additional information for this use case? I'm missing some pieces in order to move forward with a proposal.

Are you referring to making changes to the element called, "Postsecondary Program Level" (global ID 001616)? The way I'm reading your request, there needs to be more options sets for this element. Is that correct? The current option sets include the following:

Major
Minor
Certificate
Other

Is this use case suggesting the addition of the option sets "credentials" and "programs"?

Any clarification you can provide would be helpful.

Thanks!
Angela
Stakeholder Engagement Specialist
QIP

**Comment by CaliStringer** [comment]
@angelajubinvilleqip That is correct.  This is an update to the existing element 001616. We would like to add additional option sets for Credentials and Programs.

**Comment by angelajubinvilleqip** [comment]
Hi @CaliStringer 

Thanks for getting back to me. In order to create a proposal to add the option sets "Credentials" and "Programs", we would need a definition for each.

The definitions for the current option sets follow the same pattern of "[Option Set] is the concentration of study for the postsecondary program". Does it capture the meaning of what you/your group needs conveyed to say, "Credentials is the concentration of study for the postsecondary program" and/or "Programs is the concentration of study for the postsecondary program" (i'm guessing that second one wouldn't really work because you're using the term to define the term)?

Your guidance on defining these proposed option sets would be most helpful.

Best,
Angela
